src/_util.scss

```diff
@@ -0,0 +1,19 @@
+.margin-top-10 {
+  margin-top: 10px !important;
+}
+
+.margin-top-20 {
+  margin-top: 20px;
+}
+
+.margin-bottom-10 {
+  margin-bottom: 10px;
+}
+
+.margin-bottom-0 {
+  margin-bottom: 0 !important;
+}
+
+.margin-left-10 {
+  margin-left: 10px !important;
+}
```

Comment:
The company I work for has stylesheets that give utilities, layouts, and other commonly needed css. I think it helps reduce the amount of custom css classes and the need to put margin-top: x in all these different component scss files as part of some custom class definition.